frontend/src/pages/SignIn.global.scss

```diff
@@ -16,4 +16,10 @@ $color-box-secondary: #DFDFDF;
 
 .header {
     color: $color-primary;
+    size: 45px;
```

Comment:
Hey, we have 2 .scss files that are 99% identical :D

Reply:
Yeah, you're right :D we need to discuss about css files in a meeting